cmd/buildah/config.go

```diff
@@ -236,9 +243,16 @@ func updateConfig(builder *buildah.Builder, c *cobra.Command, iopts configResult
 		for _, labelSpec := range iopts.label {
 			label := strings.SplitN(labelSpec, "=", 2)
 			if len(label) > 1 {
-				builder.SetLabel(label[0], label[1])
-			} else {
+				if strings.HasSuffix(label[1], "-") {
```

Comment:
Ditto, I don't think you need this section  Just check if the env[0] ends in "-"

Reply:
Okay, I'll change it to that.